lib/crypto.js

```diff
@@ -17,13 +18,18 @@ const getFipsCrypto = binding.getFipsCrypto;
 const setFipsCrypto = binding.setFipsCrypto;
 
 const Buffer = require('buffer').Buffer;
-const constants = require('constants');
 const stream = require('stream');
 const util = require('util');
 const LazyTransform = require('internal/streams/lazy_transform');
 
 const DH_GENERATOR = 2;
 
+Object.defineProperty(exports, 'constants', {
```

Comment:
To intentionally make `constants` read-only. I want to avoid anyone doing something like `crypto.constants = { /* something else */ }`